extensions/physics-nodes/CCPhysicsSprite.cpp

```diff
@@ -393,22 +393,20 @@ void PhysicsSprite::syncPhysicsTransform() const
 #endif
 }
 
-// returns the transform matrix according the Chipmunk Body values
-const Mat4& PhysicsSprite::getNodeToParentTransform() const
+void PhysicsSprite::onEnter()
 {
-    syncPhysicsTransform();
-    
-	return _transform;
+    Node::onEnter();
+    scheduleUpdate();
```

Comment:
The `syncPhysicsTransform` method should be executed after the other scheduled callbacks is executed. Scheduler does not ensure it.